src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1920,7 +1920,7 @@ private async Task<bool> RunTestWorkflow(IStorageAccount storageAccount, bool us
         private static void WriteGeneralRetryMessageToConsole()
             => ConsoleEx.WriteLine("Please try deployment again, and create an issue if this continues to fail: https://github.com/microsoft/CromwellOnAzure/issues");
 
-        private async Task<bool> TestWorkflowAsync(IStorageAccount storageAccount, bool usePreemptibleVm = true)
+        private async Task<bool> TestWorkflowAsync(IStorageAccount storageAccount, bool usePreemptibleVm = true, string UbuntuImageName)
```

Comment:
Should be `private async Task<bool> TestWorkflowAsync(IStorageAccount storageAccount, bool usePreemptibleVm = true, string UbuntuImageName = default)`